codegen/lib/graphql_java_gen.rb

```diff
@@ -274,4 +274,34 @@ def abstract_types
       end
     end
   end
+
+  def java_doc(element)
+    doc = ''
+    unless element.description.nil?
+      description = wrap_text(element.description, 100)
```

Comment:
usually from best practises the max line length on java supposed to be 100 - 140. So it's not specific to Shopify only